### ISSUE TITLE
[A11y] New item in bulleted list doesn't use custom announcement

### ISSUE THREAD

**Issue by vhuseinova-msft** [microsoft/roosterjs#2733]
**Describe the bug**
RoosterJS provides a way to set a custom announcements for new items bulleted list (`announcerStringGetter`). But it looks like even when `announcerStringGetter` returns a value for `announceListItemBullet`, it is still not used by announcer.

**To Reproduce**
1. Open Rooster JS demo web site
2. Click add bulleted list button
3. type some text
4. Press enter
5. Observe that Narrator/VoiceOver doesn't announce a custom announcement for  `announceListItemBullet`
-------
1. Open Rooster JS demo web site
2. Press "-" key and space key
3. type some text
4. Press enter
5. Observe that Narrator/VoiceOver doesn't announce a custom announcement for  `announceListItemBullet`

**Expected behavior**
The `announceListItemBullet` string returned in `announcerStringGetter` should be used to announce when a new bullet is added (in autoformatted list and when list is added with `toggleBullet` api)

**Device Information**
 - OS: [e.g. iOS] MacOS
 - Browser [e.g. chrome, safari] Edge

**Comment by vhuseinova-msft** [comment]
cc @BryanValverdeU
